src/layout/header/Logo.tsx

```diff
@@ -1,13 +1,33 @@
-import { Box, Button, Typography } from '@mui/material';
+import { Box, Button, Theme, Typography, useMediaQuery } from '@mui/material';
+import { useTranslation } from 'react-i18next';
 import { Link as RouterLink } from 'react-router-dom';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
-export const Logo = () => (
-  <Box sx={{ gridArea: 'logo' }}>
-    <Button data-testid="logo" component={RouterLink} to={UrlPathTemplate.Home}>
-      <Typography variant="h5" component="span" sx={{ color: 'white', fontWeight: 900, fontSize: '3rem' }}>
-        NVA
-      </Typography>
-    </Button>
-  </Box>
-);
+export const Logo = () => {
+  const { t } = useTranslation();
+  const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('md'));
```

Comment:
Noen skjermstørrelser som ikke får plass til header nå, så jeg tenker kanskje å skjule "Nasjonalt Vitenarkiv" enda tidligere?
```suggestion
  const showFullAppName = useMediaQuery((theme: Theme) => theme.breakpoints.down('lg'));
```
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/fef87f9d-4903-4ec2-a029-620f36ed38d2)